### PR TITLE
Allow running smoke tests locally on windows

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

File: smoke-tests/src/test/groovy/io/opentelemetry/smoketest/CrashEarlyJdk8Test.groovy [hunk 1]
```diff
@@ -17,10 +16,14 @@ import org.testcontainers.utility.MountableFile
 import spock.lang.IgnoreIf
 import spock.lang.Specification
 
+import java.time.Duration
```
Comment (anuraaga) [inline smoke-tests/src/test/groovy/io/opentelemetry/smoketest/CrashEarlyJdk8Test.groovy:19]: Formatting looks off

**Review by anuraaga — APPROVED** [review]

File: smoke-tests/src/test/groovy/io/opentelemetry/smoketest/CrashEarlyJdk8Test.groovy (hunk 1 above)
Comment (trask) [reply smoke-tests/src/test/groovy/io/opentelemetry/smoketest/CrashEarlyJdk8Test.groovy:19]: I think this is the correct formatting (#1708)

File: smoke-tests/src/test/java/io/opentelemetry/smoketest/TestContainerManager.java [hunk 2]
```diff
@@ -28,4 +28,9 @@
       String[] command);
 
   void stopTarget();
+
+  static boolean useLinuxContainers() {
```
Comment (iNikem) [inline smoke-tests/src/test/java/io/opentelemetry/smoketest/TestContainerManager.java:32]: This method's result is always reverted, should we revert the method itself instead?

Comment (trask) [reply smoke-tests/src/test/java/io/opentelemetry/smoketest/TestContainerManager.java]: makes sense 👍

**Pull request merged**